updated readme to mention verilator dependency (#1984)
chipsalliance/chisel3: README.md

cd chisel3
sbt compile
```

> In order to run the following unit tests, make sure you have Verilator installed and loaded
> (Check this by running 'which verilator')

Reviewer: ```suggestion

In order to run the following unit tests, make sure you have Verilator installed and loaded
(you can check this by running 'which verilator').
```

Reviewer 2: ```suggestion
In order to run the following unit tests, make sure you have Verilator installed and on your `PATH`
(Check this by running `which verilator`).
```

Reviewer 3: I like Jack's suggestion better

Reviewer: i think they can both be used together; i'm just suggesting that the capitalization be fixed in the parentheses.

Reviewer 2: I've incorporated your change into mine.
